tests/call_run_on_script_with_keras_fit.py

```diff
@@ -18,16 +18,31 @@
 from tensorflow_cloud import machine_config
 from tensorflow_cloud import run
 
+# # Automated CPU strategy: CPU chief config and no workers
+# run.run(
```

Comment:
Why the commenting-out? Should this be a separate test?

Reply:
Yes, this is until we figure out how to run integration tests. Did not want to create a separate file for every workflow combination.